dev/io.openliberty.mcp.internal/resources/io/openliberty/mcp/internal/CWMCM.nlsprops

```diff
@@ -0,0 +1,153 @@
+CWMCM0009I.get.disallowed.explanation=A GET request was sent but this functionality is not supported.
+CWMCM0009I.get.disallowed.useraction=Use a supported HTTP request i.e POST
+
+# Used for server logging
+# {0} = tool name
+# {1} = exception details
+CWMCM0010E.internal.server.error.detailed=CWMCM0010E: The {0} tool method threw an unexpected exception. The exception was {1}
+CWMCM0010E.internal.server.error.detailed.explanation=The tool method threw an unexpected exception.
+CWMCM0010E.internal.server.error.detailed.useraction=Review the server message.log and FFDC logs to identify the problem.
+
+# Used as part of a response to a client
+CWMCM0011E.internal.server.error=CWMCM0011E: An internal server error occurred while running the tool.
+CWMCM0011E.internal.server.error.explanation=The server was not able to complete the tool call request.
+CWMCM0011E.internal.server.error.useraction=Check that the request is correct. More information about the failure may be found in the server messages.log.
+
+# Used for server logging
+# do not translate bean
+# {0} = bean error details
+# {1} = tool name
+CWMCM0012E.bean.release.fail=CWMCM0012E: An error occurred when releasing the CDI bean after calling the {1} tool. The error is: {0}
+CWMCM0012E.bean.release.fail.explanation=When calling a tool method, an instance of the CDI bean which declares the method is acquired before calling the method and released afterwards. An error occurred when releasing the bean instance.
+CWMCM0012E.bean.release.fail.useraction=Review the server message.log and FFDC logs to identify the problem.
+
+# Used for server logging and client response
+# do not translate MCP-Protocol-Version, MCP
+# {0} = List of supported MCP-Protocol-Version values
+CWMCM0013E.unsupported.mcp.http.version=CWMCM0013E: An unsupported MCP-Protocol-Version header was provided. Supported values: {0}
+CWMCM0013E.unsupported.mcp.http.version.explanation=The protocol version indicated by the MCP-Protocol-Version header is not supported by the server.
+CWMCM0013E.unsupported.mcp.http.version.useraction=Use a supported protocol version.
+
+# Used as part of a response to a client
+# {0} = Method
+# {1} = RequestURI
+# {2} = RequestQuery
+CWMCM0014E.unexpected.server.error=CWMCM0014E: Unexpected Server Error. Method={0} RequestURI={1} RequestQuery={2}
+CWMCM0014E.unexpected.server.error.explanation=An unexpected server error occurred whilst processing the request. The request could not be completed.
+CWMCM0014E.unexpected.server.error.useraction=No action is required.
+
+# Used for server logging
+# {0} = Method
+# {1} = RequestURI
+# {2} = RequestQuery
+# {3} = ExceptionMessage
+CWMCM0015E.unexpected.server.error.exception=CWMCM0015E: Unexpected Server Error. Method={0} RequestURI={1} RequestQuery={2}, Exception: {3}
+CWMCM0015E.unexpected.server.error.exception.explanation=An unexpected server error occurred whilst processing the request. The request could not be completed.
```

Comment:
- Change `whilst` to `while`
- Change `could not` to `cannot` or to `might not`